dashboard/test/controllers/api/v1/pd/applications_controller_test.rb

```diff
@@ -577,12 +577,13 @@ class ApplicationsControllerTest < ::ActionController::TestCase
       refute response_csv.first.include?(column)
     end
 
-    test 'cohort view returns applications that are accepted and withdrawn' do
+    test 'cohort view returns teacher applications of correct statuses' do
       expected_applications = []
-      (Pd::Application::ApplicationBase.statuses - ['interview']).each do |status|
+      teacher_cohort_view_statuses = Api::V1::Pd::ApplicationsController::COHORT_VIEW_STATUSES & TEACHER_APPLICATION_CLASS.statuses
```

Comment:
Ooh, is this the set intersection usage of `&`?

Reply:
yep!